test/v2-migration-tests/src/test/resources/software/amazon/awssdk/v2migrationtests/maven/before/src/main/java/foo/bar/S3_Streaming.java

```diff
@@ -0,0 +1,93 @@
+/*
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ *  http://aws.amazon.com/apache2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+package foo.bar;
+
+import com.amazonaws.services.s3.AmazonS3;
+import com.amazonaws.services.s3.AmazonS3ClientBuilder;
+import com.amazonaws.services.s3.model.CannedAccessControlList;
+import com.amazonaws.services.s3.model.PutObjectRequest;
+import com.amazonaws.services.s3.model.S3Object;
+import java.io.ByteArrayInputStream;
+import java.io.File;
+import java.io.InputStream;
+
+public class S3_Streaming {
+
+    AmazonS3 s3 = AmazonS3ClientBuilder.defaultClient();
+
+    void getObject(String bucket, String key) throws Exception {
+        S3Object s3Object = s3.getObject(bucket, key);
+        s3Object.getObjectContent().close();
+    }
+
+    void putObject_bucketKeyContent(String bucket, String key, String content) {
+        s3.putObject(bucket, key, content);
+    }
+
+    void putObject_bucketKeyFile(String bucket, String key, File file) {
+        s3.putObject(bucket, key, file);
+    }
+
+    /**
+     * Mixed ordering to ensure the files are assigned correctly
+     */
+    void putObject_requestPojoWithFile(String bucket, String key) {
+        File file1 = new File("file1.txt");
```

Comment:
What happens if the files are defined out of order? something like file4 is defined first

Reply:
It will get assigned correctly. Will update tests to define file4 first